taxonomy/utils.py

```diff
@@ -203,6 +313,24 @@ def is_course_skill_blacklisted(course_key, skill_id):
     ).exists()
 
 
+def is_program_skill_blacklisted(program_uuid, skill_id):
+    """
+    Return the black listed status of a program skill.
+
+    Arguments:
+        program_uuid: uuid of the program whose skill need to be checked.
+        skill_id (int): Primary key identifier of the skill that need to be checked.
+
+    Returns:
+        (bool): True if program-skill (identified by the arguments) is black-listed, False otherwise.
+    """
+    return ProgramSkill.objects.filter(
+        program_uuid=program_uuid,
+        skill_id=skill_id,
+        is_blacklisted=True,
```

Comment:
Is there some other flow for the blacklisting i.e. false positives ?

Reply:
I haven't written any custom flow for it. The existing util method for course `blacklist_course_skill` isn't being used anywhere so I assumed that the skills can only be blacklisted through django admin.